src/components/pages/ParentHome/index.js

```diff
@@ -5,11 +5,10 @@ import ParentSidebar from './ParentSidebar';
 import { dummyData } from '../../../dummyData';
 
 function ParentHome() {
-  const [parentData, setParentData] = useState(dummyData);
   const [scheduleData, setScheduleData] = useState(dummyData);
   return (
     <div className="App">
-      <ChildrenList children={parentData}></ChildrenList>
+      <ChildrenList></ChildrenList>
```

Comment:
Very good work. Another thing - probably just preference - is we can just make the components self-closing since we aren't going to be passing in props.

Reply:
Gotcha! I just made those changes and they definitely look much cleaner now.